src/sdk/conference/src/main/java/owt/conference/SubscriptionCapabilities.java

```diff
@@ -22,9 +22,9 @@
 import java.util.List;
 
 /**
- * Capabilities for subscribing a RemoteStream, which indicates the video or/and audio options
- * that ConferenceClient may use to subscribe a RemoteStream. Subscribing a RemoteStream with the
- * options that beyonds its SubscriptionCapabilities may cause failure.
+ * Capabilities for subscribing a RemoteStream besides PublicationSettings, which indicates the video or/and audio
```

Comment:
> besides PublicationSettings

This is what **extra** means. For `SubscriptionCapabilities` I think it could keep the old description.